ota-plus-web/app/reactapp/src/pages/Home/index.jsx

```diff
@@ -0,0 +1,98 @@
+/** @format */
+
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useObserver } from 'mobx-react';
+import { useStores } from '../../stores/hooks';
+import { MetaData, FadeAnimation } from '../../utils';
+import { SanityCheckContainer, Terms } from '../../containers';
+import { Loader } from '../../partials';
+import DashboardStepper from '../../components/dashboard/new/DashboardStepper';
+import { HomeWrapper } from './styled';
+
+const title = 'Home';
+
+function useStoreData() {
+  const { stores } = useStores();
+  return useObserver(() => ({
+    campaignsStore: stores.campaignsStore,
+    devicesStore: stores.devicesStore,
+    groupsStore: stores.groupsStore,
+    softwareStore: stores.softwareStore,
+    updatesStore: stores.updatesStore,
+    provisioningStore: stores.provisioningStore,
+    sanityCheckCompleted: stores.provisioningStore.sanityCheckCompleted,
+    userStore: stores.userStore
+  }));
+}
+
+const Home = ({ uiAutoFeatureActivation, uiUserProfileMenu }) => {
+  const {
+    campaignsStore,
+    devicesStore,
+    groupsStore,
+    softwareStore,
+    updatesStore,
+    provisioningStore,
+    sanityCheckCompleted,
+    userStore,
+  } = useStoreData();
+
+  useEffect(() => {
+    if (!uiAutoFeatureActivation) {
+      provisioningStore.sanityCheckCompleted = true;
+    }
+    if (uiUserProfileMenu) {
+      provisioningStore.namespaceSetup();
+    }
+    devicesStore.fetchDevicesStats();
+    devicesStore.fetchNotSeenRecentlyDevices();
+    // devicesStore.fetchUngroupedDevices();
```

Comment:
Please remove comments if we are not using them.

Reply:
It was temporary change, just for testing.